tests/sqlparser_bigquery.rs

```diff
@@ -632,12 +638,12 @@ fn parse_typed_struct_syntax() {
         expr_from_projection(&select.projection[1])
     );
 
-    let sql = r#"SELECT STRUCT<DATE>("2011-05-05"), STRUCT<DATETIME>(DATETIME '1999-01-01 01:23:34.45'), STRUCT<FLOAT64>(5.0), STRUCT<INT64>(1)"#;
-    let select = bigquery().verified_only_select(sql);
+    let sql = r#"SELECT STRUCT<DATE>('2011-05-05'), STRUCT<DATETIME>(DATETIME '1999-01-01 01:23:34.45'), STRUCT<FLOAT64>(5.0), STRUCT<INT64>(1)"#;
```

Comment:
change " to ', so it is understandable by generic dialect also

Reply:
Rather than updating the existing test, can you please keep the existing test (that uses `"`) and add new coverage for bigquery_and_generic dialect (with the `'`)?

This is to avoid losing existing test coverage

Reply:
i added extra test, was thinking of using a scenario struct to reuse the query logic and just replacing " with ' in the middle before we invoke, but i guess a bit boilerplating is easier to read

Reply:
Yeah, thank you -- I know it is a lot of repetition but it is easier to understand I think. Thank you 🙏